README.md

```diff
@@ -148,28 +153,78 @@ One may specify multiple data locations by a list of locations as long as they h
   ```
   location: ["./data/sample.parquet", "https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2022-01.parquet"]
   ```
-- `using`: string. Required.  
-One of `parquet`, `spark_sql` and `delta`, or the custom file format specified by the
-loader function in `steps/ingest.py` (ex. `csv`).
 
 
-- `loader_method`: string. Optional.  
-Method name of the custom loader function from `steps/ingest.py`.  
-<u>Example</u>: 
+</details>
+
+<details>
+<summary><strong><u>Using: "delta"</u></strong></summary>
+
+- `location`: string. Required.  
+Dataset locations or the catalog path
+<u>Examples</u>:
+  ```
+  location: catalog.schema.table
   ```
-  loader_method: load_file_as_dataframe
+  ```
+  location: "dbfs:/user/hive/warehouse/ml.db/all_prices_1990_2021_mar_csv"
   ```
 
-- `sql`: string. Required if format is `spark_sql`.  
-Specifies a SparkSQL statement that identifies the dataset to use.
+- `version`: int. Optional.  
+Use this to specify the Delta table version to read from.
 
 
-- `version`: int. Optional.  
-If the `delta` format is specified, use this to specify the Delta table version to read from.
+- `timestamp`: timestamp. Optional.
+Use this to specify the timestamp at which to read data.
+</details>
 
+<details>
+<summary><strong><u>Using: "spark_sql"</u></strong></summary>
 
-- `timestamp`: timestamp. Optional.  
-If the `delta` format is specified, use this to specify the timestamp at which to read data.
+- `location`: string. Optional.
+Dataset locations or the catalog path. If location is specified then automatically all the rows and columns are ingested without any selection. Either location or sql must be specified
+<u>Examples</u>:
+  ```
+  location: catalog.schema.table
+  ```
+  ```
+  location: "dbfs:/user/hive/warehouse/ml.db/all_prices_1990_2021_mar_csv"
+  ```
+
+- `sql`: string. Optional.  
+Specifies a SparkSQL statement that identifies the dataset to use. Either location or sql must be specified
+<u>Examples</u>:
+  ```
+  sql: SELECT * FROM delta.`dbfs:/databricks-datasets/nyctaxi-with-zipcodes/subsampled`
+  ```
+
+</details>
+
+<details>
+<summary><strong><u>Using: "custom"</u></strong></summary>
+
+- `location`: string. Required.  
+Dataset locations on the local filesystem are supported, as 
+well as HTTP(S) URLs and any other remote locations [resolvable by MLflow](https://mlflow.org/docs/latest/tracking.html#artifact-stores).
+One may specify multiple data locations by a list of locations as long as they have the same data format (see example below)
+<u>Examples</u>:
+  ```
+  location: ./data/sample.csv
+  ```
+  ```
+  location: https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2022-01.parquet
+  ```
+  ```
+  location: ["./data/sample.parquet", "https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2022-01.parquet"]
```

Comment:
Prefer not using `parquet` format for custom method. `csv` is probably sufficient.